### PR TITLE
added helm directory

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/giantswarm/issues/2334.

### CODE REVIEW

File: helm/node-operator-chart/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+          mountPath: /var/run/node-operator/configmap/
+        livenessProbe:
+          httpGet:
+            path: /healthz
+            port: 8000
+          initialDelaySeconds: 30
+          timeoutSeconds: 1
+        resources:
+          requests:
+            cpu: 100m
```
Comment (JosephSalisbury) [inline helm/node-operator-chart/templates/deployment.yaml]: I would make this the same as requests, for that guaranteed foo

Comment (xh3b4sd) [reply helm/node-operator-chart/templates/deployment.yaml]: Why the magic? I just copied this as it is from kvm-operator. Does that mean we should change this everywhere?

Comment (xh3b4sd) [reply helm/node-operator-chart/templates/deployment.yaml]: Let me know when this is important. We can change later. I want to move now.

Comment (JosephSalisbury) [reply helm/node-operator-chart/templates/deployment.yaml]: See https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed

If requests == limits, kubernetes can do better with scheduling. If this is the case for kvm-operator too, we can fix it there.

Comment (xh3b4sd) [reply helm/node-operator-chart/templates/deployment.yaml]: I still miss something. What is this doing? What is guaranteed? I couldn't find this info in the docs. What problem does this solve?

Comment (JosephSalisbury) [reply helm/node-operator-chart/templates/deployment.yaml]: https://medium.com/google-cloud/quality-of-service-class-qos-in-kubernetes-bb76a89eb2c6 is a good write up

Comment (xh3b4sd) [reply helm/node-operator-chart/templates/deployment.yaml]: Thanks. Will read later.

**Pull request merged**